dockerversion/version_lib.go

```diff
@@ -6,7 +6,6 @@ var (
 	GitCommit             = "library-import"
 	Version               = "library-import"
 	BuildTime             = "library-import"
-	IAmStatic             = "library-import"
```

Comment:
ISTR we kept this field in `dockerversion` with the intent to (potentially) surface this information in `docker version` / `docker info` and/or `--version` output to help debugging issues when users report issues (it would save us to ask "is this a static binary, or did you install from deb/rpm?").

I don't think we ever did this, but perhaps still something to consider.